fs-api/fs-api.cabal

```diff
@@ -54,7 +54,7 @@ library
     , digest
     , directory   >=1.3  && <1.4
     , filepath    >=1.4  && <1.5
-    , io-classes  ^>=0.3
+    , io-classes  ^>=0.5
```

Comment:
Why not `>=0.3 && < 0.6`?

Reply:
And when we compare the new bounds to https://github.com/input-output-hk/cardano-haskell-packages/pull/187, are we not too strict?